### PR TITLE
Fix ASR issues and enable reprompt

### PR DESCRIPTION
**Pull request by rogerkibbe**
Fixes ASR issues with letters A, B, C, D e.g. Bixby may interpret "C" as "See" - this is now handled.
If the user does not provide an understood answer e.g. one of the letters or an option, now reprompts with the question and encourages user to use letters for the answer. Boolean turns this behavior on/off

### CODE REVIEW

File: code/UpdateQuiz.js [hunk 1]
```diff
@@ -1,35 +1,95 @@
-const { buildQuestionToSpeak } = require("./lib/util");
+const util = require("./lib/util");
 
-module.exports.function = function updateQuiz (quiz, answer) {
+// Will repromt if the user does not give a valid input. Set to false to not reprompt
+// Futue enhancement would be to reprompt only n times and then give up (suggest n = 1 or 2)
+const reprompt = true;
+
+// ASR Aliases to handle ASR variations for "A", "B" "C" or "D" etc
```
Comment (siamakh) [inline code/UpdateQuiz.js]: Embarrassing that we have to add these here. If this is a common pattern we should make sure ASR gets it correctly. Or we can change to using numerics.

File: resources/base/views/UpdateQuiz.view.bxb [hunk 2]
```diff
@@ -1,52 +1,63 @@
...
-          }
-        }
-        on-submit {
-          goal: Answer
-          value: viv.core.FormElement(answer)
-        }
-      }
-    }
-  }
+	match: Answer(this) {
```
Comment (siamakh) [inline resources/base/views/UpdateQuiz.view.bxb]: 2 character is the default indentation for our capsule source files. Can we indent this with the IDE?

**Review by siamakh — COMMENTED** [review]
I am not happy with how us trying to recover from ASR mistakes in the capsule. This is not a good example for devs to follow.

File: code/UpdateQuiz.js (hunk 1 above)
Comment (rogerkibbe) [reply code/UpdateQuiz.js]: Going to add comment to indicate this is a temporary workaround. Numerics cause too many issue with quiz answers which are numbers.

**Pull request closed without merging**